packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -769,12 +773,14 @@ const ConstraintEditor = observer(
 
 const SuperTypeEditor = observer(
   (props: {
+    editorState: ClassEditorState;
```

Comment:
I don't think we need this change

Reply:
You're right ^^;  Deleted and just called store